src/main/java/seedu/simplykitchen/logic/commands/EditCommand.java

```diff
@@ -90,8 +93,9 @@ private static Food createEditedFood(Food foodToEdit, EditFoodDescriptor editFoo
         Description updatedDescription = editFoodDescriptor.getDescription().orElse(foodToEdit.getDescription());
         Priority updatedPriority = editFoodDescriptor.getPriority().orElse(foodToEdit.getPriority());
         ExpiryDate updatedExpiryDate = editFoodDescriptor.getExpiryDate().orElse(foodToEdit.getExpiryDate());
+        Quantity updateQuantity = editFoodDescriptor.getQuantity().orElse(foodToEdit.getQuantity());
         Set<Tag> updatedTags = editFoodDescriptor.getTags().orElse(foodToEdit.getTags());
-        return new Food(updatedDescription, updatedPriority, updatedExpiryDate, updatedTags);
+        return new Food(updatedDescription, updatedPriority, updatedExpiryDate, updateQuantity, updatedTags);
```

Comment:
Using "updatedQuantity" will make it consistent with the naming of the other variables.